DQM/Physics/src/CentralityDQM.cc

```diff
@@ -141,8 +141,10 @@ void CentralityDQM::analyze(const edm::Event& iEvent,
 
   //  if (!cent.isValid()) return;
   if(cent.isValid()){
-  int hibin = *cbin;
-  
+  int hibin = -999;
+  if(cbin.isValid()) hibin = *cbin;
+  else edm::LogWarning("CentralityDQM") << "invalid collection: centralityBin " << std::endl;
```

Comment:
what is the reason to pollute the logs with this message by default?
The same question applies to other cases.

If the goal is to keep a reminder where it should be added again, perhaps add these printouts with a compile flag  e.g. ```#ifdef CENTRALITY_WARNING ```